plugins/scsi/fu-scsi-device.c

```diff
@@ -46,9 +70,36 @@ fu_scsi_device_probe(FuDevice *device, GError **error)
 				    "no assigned vendor");
 		return FALSE;
 	}
+
 	vendor_id = g_strdup_printf("SCSI:%s", fu_device_get_vendor(device));
 	fu_device_add_vendor_id(device, vendor_id);
 
+	/* the controller could really on any bus.. */
```

Comment:
could really be*

 May be word it as a TODO to add better search for ufshci?

Reply:
Agree, fixed.